app/models/concerns/orchestration/dhcp.rb

```diff
@@ -19,6 +20,20 @@ def dhcp_record
     @dhcp_record ||= (provision? && jumpstart?) ? Net::DHCP::SparcRecord.new(dhcp_attrs) : Net::DHCP::Record.new(dhcp_attrs)
   end
 
+  def rebuild_dhcp
+    begin
+      if dhcp?
+        set_dhcp
```

Comment:
Should we also run removal methods (e.g. del_dhcp) in these methods? Perhaps throwing away the result?  I was just trying to test it with a host that already had its entries, and of course it fails.  I'm not sure if that'd be expected, a user would instead probably expect it to destroy/create.